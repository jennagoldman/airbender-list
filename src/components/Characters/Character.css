.characterName {
  font-family: 'Lemonada', sans-serif;
}

li div {
  background: #fff;
  width: 300px;
  height: 300px;
  padding: 25px 10px;
  border-radius: 15px;
  box-shadow: 10px 10px #000;
}

li div p {
  margin: 10px;
  font-size: 24px;
}

li a {
  color: coral;
  text-shadow: 1px 1px #000;
}

li div img {
  width: 250px;
  height: 250px;
  border-radius: 10px;
}