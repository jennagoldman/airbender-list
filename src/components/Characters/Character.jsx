import React from 'react';
import PropTypes from 'prop-types';
import styles from './Character.css';

const Character = ({ name, imageUrl }) => (
  <div className={styles.characterDiv}>
    <p className={styles.characterName}>{name}</p>
    <img src={imageUrl} alt={name} />
  </div>
);

Character.propTypes = {
  name: PropTypes.string.isRequired,
  imageUrl: PropTypes.string,
};


export default Character;
