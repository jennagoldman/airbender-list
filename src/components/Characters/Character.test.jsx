import React from 'react';
import { shallow } from 'enzyme';
import Character from './Character.jsx';

describe('Character', () => {
  it('matches a snapshot', () => {

    const wrapper = shallow(<Character name="Momo" imageUrl="https://vignette.wikia.nocookie.net/avatar/images/4/43/Inquisitive_Momo.png/revision/latest?cb=20081225191217" />);
    expect(wrapper).toMatchSnapshot();
  });
});
