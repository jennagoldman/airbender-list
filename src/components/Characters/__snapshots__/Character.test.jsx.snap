// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character matches a snapshot 1`] = `
<div
  className="characterDiv"
>
  <p
    className="characterName"
  >
    Momo
  </p>
  <img
    alt="Momo"
    src="https://vignette.wikia.nocookie.net/avatar/images/4/43/Inquisitive_Momo.png/revision/latest?cb=20081225191217"
  />
</div>
`;
